src/main/scala/mesosphere/chaos/chaos/ChaosModule.scala

```diff
@@ -0,0 +1,19 @@
+package mesosphere.chaos.chaos
+
+import mesosphere.chaos.AppConfiguration
+import mesosphere.chaos.http.{ HttpModule, HttpConf }
+import mesosphere.chaos.metrics.MetricsModule
+import org.rogach.scallop.ScallopConf
+
+/**
+  * Created by alex on 16/02/16.
+  */
+class ChaosModule(args: Seq[String]) {
```

Comment:
You require a scallop config, rather than creating one in the module. Subsequent modules will extend this config.